pages/mesosphere/dcos/1.14/administering-clusters/backup-and-restore/zookeeper/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: layout.pug
+navigationTitle: ZooKeeper State
+title: Back up and Restore ZooKeeper
+menuWeight: 1
+excerpt: Backing up and restoring DC/OS ZooKeeper state
+render: mustache
+model: /mesosphere/dcos/1.14/data.yml
+--- 
+
+Back up the state of ZooKeeper inside a DC/OS cluster, and later restore from that backup.
+
+You may wish to back up the ZooKeeper state of your cluster before performing an upgrade or downgrade. You may need to restore your cluster to a known good state if something goes wrong during an upgrade. It is highly recommended to backup ZooKeeper state regularly to be prepared for the worst-case scenario.
```

Comment:
Backing up ZooKeeper will allow you to return a cluster to a known good state. As such, it is highly recommended to backup ZooKeeper state regularly to be prepared for the worst-case scenario. When performing maintenance operations, such as an upgrade or downgrade, you may wish to back up the ZooKeeper state before beginning the maintenance.